test/unit/tag_test.rb

```diff
@@ -92,4 +92,29 @@ class TagTest < ActiveSupport::TestCase
     assert !node(:blog).responses('replication').empty?
     assert node(:blog).response_count('replication') > 0
   end
+
+  test "returns empty array if users are  following both the given tags and this tag" do
+    tag = tags(:spam)
+    given_tags = [tags(:chapter)]
+    assert_equal [], tag.followers_who_dont_follow_tags(given_tags)
+  end
+
+  test " returns users following this tags but not given tags" do
+    # users following tag are bob, unbanned_spammer, admin
+    # users following tag1 are bob, ubanned_spammer, moderator
+    # users following tag2 are spammer, newcomer
+    tag = tags(:test)
+    tag1 = tags(:awesome)
```

Comment:
We also need to add something to ensure `tag.subscriptions` (alias of `tag.tag_selection`) includes the extra condition:

`.where(following: true)`

https://github.com/publiclab/plots2/blob/master/app/models/tag.rb#L42

I think the extra fixtures you added aren't affecting the test here, since if they were, the test should fail until we add the extra `following: true` condition. Right? 

Thank you!